src/component/mxgraph/Options.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export enum FitType {
```

Comment:
:warning: I would move this to the BpmnVisualization file. The type is part of the public api and is not related to mxgraph

Reply:
If it has been placed in mxgraph because of a circular dependency, let's create a options.ts file at the src root and let's put all options types there.

Reply:
Originally, I put it in the **BpmnVisualization** file, but it generated circular dependencies.

Reply:
so let's put it in a dedicated file :smile:

Reply:
Done 😉